Formula/lgogdownloader.rb

```diff
@@ -0,0 +1,46 @@
+class Lgogdownloader < Formula
+  desc "Open source downloader for GOG.com games"
+  homepage "https://sites.google.com/site/gogdownloader"
+  url "https://sites.google.com/site/gogdownloader/lgogdownloader-3.2.tar.gz"
+  sha256 "9e60af50c18533884ca317778b291152d5f04d93dfd3dc2579f6f965538b2031"
+
+  depends_on "cmake" => :build
+  depends_on "help2man" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "boost"
+  depends_on "htmlcxx"
+  depends_on "jsoncpp"
+  depends_on "liboauth"
+  depends_on "rhash"
+  depends_on "tinyxml2"
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", "-DCMAKE_INSTALL_PREFIX=#{prefix}",
```

Comment:
What's with not just using `std_cmake_args` here? Does that break something?

Reply:
Thanks @DomT4 for pointing this out. As you suggested, the additional flags from `std_cmake_args` don’t seem to break anything. I have now replaced the entire statement with this:

```
system "cmake", "..", *std_cmake_args
```